hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/HoodieSparkSqlTestBase.scala

```diff
@@ -140,8 +140,11 @@ class HoodieSparkSqlTestBase extends FunSuite with BeforeAndAfterAll {
       spark.sql(sql)
     } catch {
       case e: Throwable =>
-        assertResult(true)(e.getMessage.contains(errorMsg))
-        hasException = true
+        if (e.getMessage.contains(errorMsg)) {
```

Comment:
Why changing this?

Reply:
It makes the unit test eaiser to debug. I was seeing some issues with unittests but previous code only gives out limited info

Reply:
Nit: I'd suggest to squash match w/ conditional: 

```
match {
  case e: Throwable if e.getMessage.contains(...) => ...
  case _ => fail(...)
}
```